### PR TITLE
Add Message#mentionsEveryone

### PR DESCRIPTION
**Pull request by Tea-Ayataka**
I really needed this

### CODE REVIEW

**Comment by Kaleidox00** [comment]
Proposal:
Not use this method but instead make the result of Message#getUserMentions contain some static "@everyone" Role, which has to be static, so it can be checked easily with #contains. I might make a PR example of what I mean.

**Comment by Tea-Ayataka** [comment]
There is no way to check if the user had 'Mentions Everyone' permission or not. So i think this PR is not useless.

**Comment by Kaleidox00** [comment]
I don't understand?

**Comment by Tea-Ayataka** [comment]
Nevermind. I'll edit this PR because this method is still useful.

**Comment by Vampire** [comment]
Please do not merge `v_3` into your PR branch, but rebase and force-push your PR branch instead.

**Comment by Tea-Ayataka** [comment]
Is this all right?

**Comment by Vampire** [comment]
Regarding merge commit, yes, thanks. I didn't review the content thoroughly though.

File: javacord-core/src/main/java/org/javacord/core/util/handler/message/MessageUpdateHandler.java [hunk 1]
```diff
@@ -120,8 +120,10 @@ public void handle(JsonNode packet) {
 
             MessageEditEvent editEvent = null;
             if (packet.has("edited_timestamp") && !packet.get("edited_timestamp").isNull()) {
-                message.ifPresent(msg ->
-                        msg.setLastEditTime(OffsetDateTime.parse(packet.get("edited_timestamp").asText()).toInstant()));
+                message.ifPresent(msg -> {
+                    msg.setLastEditTime(OffsetDateTime.parse(packet.get("edited_timestamp").asText()).toInstant());
+                    msg.setMentionsEveryone(packet.get("mention_everyone").asBoolean());
```
Comment (Bastian) [inline javacord-core/src/main/java/org/javacord/core/util/handler/message/MessageUpdateHandler.java]: As far as I know this value never updates.

Comment (Bastian) [reply javacord-core/src/main/java/org/javacord/core/util/handler/message/MessageUpdateHandler.java]: Ok, I just tested it and it indeed updates.
My bad.

**Comment by Bastian** [comment]
Thank you :)

**Pull request merged**